views/pages/organisation/index.html

```diff
@@ -11,6 +11,29 @@
     <div class="govuk-grid-column-full">
         {% block mainContent %}
                 <h1 id="header" data-page="{{hmpoPageKey}}">{{ translate("pages." + hmpoPageKey + ".h1", { default: hmpoTitle }) | safe }}: {{orgInfo.name}}</h1>
+                <h3>Organisation Team(s)</h3>
+                {% set teamHeaderArray = [] %}
+                {% set teamsArray = [] %}
+                {% for item in ["Team", "Users"] %}
+                    {% set teamHeaderArray = (teamHeaderArray.push( { text: item }), teamHeaderArray) %}
+                {% endfor %}
+
+                {% for team in teams | sort('team.name') %}
+                    {% set rowArray = [] %}
+                    {% set teamLink = "/admin/organisations/" + orgInfo.id + "/teams/" + team.id %}
+                    {% set rowArray = (rowArray.push( { html: "<a href=" + teamLink +">" + team.name + "</a>" } ), rowArray) %}
```

Comment:
Do we need `rowArray` on lines 24 and 25 here. Will the `rowArray.push` not just update it?

Reply:
I tried with just the push command however it seems like nunjucks doesn't have proper documentation how it should be done. Everyone suggesting this way so thats why no other option for this